test/jwt_test.dart

```diff
@@ -115,6 +115,9 @@ void main() {
       expect(parsedToken.issuer, equals('https://foobar.com'));
       expect(parsedToken.getClaim('pld'), equals('payload'));
       expect(parsedToken.getClaim('map'), equals({'key': 'value'}));
+      
+      expect(parsedToken.claims['pld'], equals('payload'));
+      expect(parsedToken.claims['map'], equals({'key': 'value'}));
```

Comment:
Thanks for adding tests! Could you also add a case which checks that modifications are not reflected in the token (re: my other comment).

Reply:
sure thing!